packages/suite-desktop/src/preload/extensions.ts

```diff
@@ -98,10 +97,20 @@ export async function getExtensions(rootFolder: string): Promise<DesktopExtensio
       const packagePath = pathJoin(extensionRootPath, "package.json");
       const packageData = await readFile(packagePath, { encoding: "utf8" });
       const packageJson = JSON.parse(packageData) as ExtensionPackageJson;
+      const readmePath = pathJoin(extensionRootPath, "README.md");
+      const readme =
```

Comment:
Since README.md and CHANGELOG.md are optional, this code can be simplified and made more robust:
- Avoid redundancy: readFile is currently called twice if the file exists.
- Improve error handling: Wrapping the read in a try-catch ensures we gracefully handle missing or unreadable files.

Suggested refactor:

```suggestion
const safeReadFile = async (filePath: string): Promise<string> => {
  try {
    return await readFile(filePath, { encoding: "utf8" });
  } catch {
    return "";
  }
};
const [readme, changelog] = await Promise.all([
  safeReadFile(readmePath),
  safeReadFile(changelogPath),
]);
```

Reply:
Suggestion implemented!